ow_lander/src/ow_lander/actions.py

```diff
@@ -1139,7 +1136,7 @@ def _is_position_in_sample_dock(self, position):
     # NOTE: This value can be found by adding the y-value on line 25 of
     #   lander_sample_dock.xacro to the point assigned to the origin on line 34
     OFFSET_RELATIVE_TO_FRAME = Point(0.0, -0.33, 0.025)
```

Comment:
Please do a separate ticket to eliminate this hardcoded transform by moving the origin of the sample dock to the center of the sample dock.

Reply:
Agreed. Here's the ticket https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-1215